applications/klighd-vscode/src/klighd-webview-reopener.ts

```diff
@@ -22,20 +22,31 @@ import { StorageService } from "./storage/storage-service";
 export class KlighdWebviewReopener {
 
     private readonly storage: StorageService
+    private toDispose: { dispose(): any }[] = []
```

Comment:
Interesting way to unregister a function from an event, had to look it up in the VS Code API to understand what is going on here. There I found that the event that the function is added to returns (in fact, any `Event`) returns a `Disposable`, so couldn't this `toDispose` field be just a `Disposable[]`?

Reply:
Ok